pkg/index/index.go

```diff
@@ -117,7 +124,11 @@ func (ti *treeIndex) Range(key, end []byte, atRev int64) (keys [][]byte, revs []
 	return keys, revs
 }
 
-func (ti *treeIndex) Tombstone(key []byte, rev Revision) error {
+func (ti *treeIndex) Tombstone(ctx context.Context, key []byte, rev Revision) error {
+	// tracing indexing component - deleting index entry (as tombstone)
```

Comment:
Tombstone is to add a flag but to delete an index entry

Reply:
ok will update comment to avoid potential misreading